### PR TITLE
docs: add package.json file diamond.js eql-triangle.js file in extra/patterns/ folder(#39515)

### PR DESCRIPTION
**Pull request by gkgbunny**
Sir please check my files and guide me if require.

### CODE REVIEW

File: extra/patterns/diamond.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+function eql(n){
+    console.log('Print a diamond pattern with 5 rows');
+    for(let i=1;i<=n;i++)
+    {
+        let a=''
+        for(let j=1;j<=n-i;j++){
```
Comment (mohit-nx) [inline extra/patterns/diamond.js]: add space between closing bracket and opening braces

File: extra/patterns/diamond.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+function eql(n){
+    console.log('Print a diamond pattern with 5 rows');
+    for(let i=1;i<=n;i++)
+    {
```
Comment (mohit-nx) [inline extra/patterns/diamond.js]: Put opening braces after closing bracket

**Comment by mohit-nx** [comment]
Please fix all the lint issues like these lint he code.

**Comment by gkgbunny** [comment]
Sir I have updated my code. Now please verify it and let me know if any further mistakes are there.

**Pull request merged**